point_pattern.py

```diff
@@ -49,7 +51,7 @@ def euclidean_distance(a, b):
     distance : float
                The Euclidean distance between the two points
     """
-    distance = None  # Add the euclidean distance algorithm here
+    distance =  ((b[0] - a[0])**2 + (b[1] - a[1])**2)**(1/2) # Add the euclidean distance algorithm here
```

Comment:
Nice job here not using math.  Be careful if you ever use Python 2.x, `(1/2) evaluates to `0`, while `(1.0/ 2)` would act as you would expect.